code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -992,16 +969,16 @@
 /datum/reagent/medicine/tricordrazine
 	name = "Tricordrazine"
 	id = "tricordrazine"
-	description = "Has a high chance to heal all types of damage. Overdose instead causes it."
+	description = "Has a decent chance to heal all types of damage. Overdose instead causes it."
 	reagent_state = LIQUID
 	color = "#C8A5DC"
 	overdose_threshold = 30
 	taste_description = "grossness"
 
 /datum/reagent/medicine/tricordrazine/on_mob_life(mob/living/M)
-	if(prob(80))
-		M.adjustBruteLoss(-1*REM, 0)
-		M.adjustFireLoss(-1*REM, 0)
+	if(prob(60))
+		M.adjustBruteLoss(-0.5*REM, 0)
+		M.adjustFireLoss(-0.5*REM, 0)
```

Comment:
Please remove the prob(80) nonsense on drugs. If you want 80% effectiveness, just lower the drug's healing power instead. RNG has no purpose there other than to fuck with people.

If you want to do that, make OD add skewium or something.

Reply:
Sure w/e